test/extended/prometheus/prometheus.go

```diff
@@ -210,9 +207,10 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			execPod := exutil.CreateCentosExecPodOrFail(oc.AdminKubeClient(), ns, "execpod", nil)
 			defer func() { oc.AdminKubeClient().CoreV1().Pods(ns).Delete(execPod.Name, metav1.NewDeleteOptions(1)) }()
 
-			tests := map[string][]metricTest{
+			tests := map[string]bool{
 				// Checking Watchdog alert state is done in "should have a Watchdog alert in firing state".
-				`ALERTS{alertname!="Watchdog",alertstate="firing"}`: {metricTest{greaterThanEqual: true, value: 1, nodata: true}},
+				// FIXME(paulfantom): ClusterMachineApproverDown and UsingDeprecatedAPIExtensionsV1Beta1 shouldn't be firing!
+				`ALERTS{alertname!~"Watchdog|ClusterMachineApproverDown|UsingDeprecatedAPIExtensionsV1Beta1",alertstate="firing"} >= 1`: false,
```

Comment:
I think the PR was merged, we can now remove the `ClusterMachineApproverDown ` from this list, correct?

And I could not see `UsingDeprecatedAPIExtensionsV1Beta1 ` firing on a recent 4.3 cluster anymore. 🤔

Reply:
I will remove them in another PR as I want this to be merged as soon as we can.

Reply:
sorry I am bit confused, is my suggestion not valid? 🤔 The tests are still running and no one approved the PR yet, so not sure why we can't do it in this PR?

Reply:
Right now, we are not testing if too many alerts are firing, that's why I want this merged as soon as it can be merged. Especially that tests which are necessary to approve this PR are flaky.

In case those 2 alerts are still firing, I don't want to debug this PR and extend the timeframe where we don't have test coverage for this situation.

I don't see anything wrong in creating a separate PR for it, do you?